django_declarative_apis/machinery/attributes.py

```diff
@@ -7,7 +7,7 @@
 
 import abc
 import collections.abc
-import logging
+import logging  # removing this causes problems
```

Comment:
What problems?

Reply:
Oh- you know what?  This is a problem caused by `import *`.  `logging` is needed in `machinery.__init__`, and it isn't imported there (but it works because it gets logging from here).

Reply:
Fixed.